ui-manchette/src/components/OperationalPointList.tsx

```diff
@@ -1,25 +1,22 @@
 import React from 'react';
 
 import OperationalPoint from './OperationalPoint';
-import { type StyledOperationalPointType } from '../types';
+import { type InteractiveWaypoint } from '../types';
 
 type OperationalPointListProps = {
-  operationalPoints: StyledOperationalPointType[];
-  activeOperationalPointId?: string;
+  waypoints: InteractiveWaypoint[];
+  activeWaypointId?: string;
 };
 
-const OperationalPointList = ({
-  operationalPoints,
-  activeOperationalPointId,
-}: OperationalPointListProps) => (
+const OperationalPointList = ({ waypoints, activeWaypointId }: OperationalPointListProps) => (
   <div className="operational-point-list ">
-    {operationalPoints.map((op) => (
+    {waypoints.map((waypoint) => (
       <div
-        key={op.id}
```

Comment:
Yay, this no longer causes duplicate `key` attributes if the same OP appears twice in the list!